docker/test/integration/steps/steps.py

```diff
@@ -256,10 +322,36 @@ def step_impl(context, cluster_name):
     cluster.set_engine("azure-storage-server")
     cluster.set_flow(None)
 
-
+@given("the kafka broker \"{cluster_name}\" is started")
+def step_impl(context, cluster_name):
+    context.test.start_single_cluster(cluster_name)
+
+@given("the topic \"{topic_name}\" is initialized on the kafka broker")
+def step_impl(context, topic_name):
+    admin = AdminClient({'bootstrap.servers': "localhost:29092"})
+    new_topics = [NewTopic(topic_name, num_partitions=1, replication_factor=1)]
```

Comment:
can we set `num_partitions` here to `3` instead of a separate `create_partitions` call?

Reply:
Good catch - I think I added the creation of new partitions "just-to-be-sure" later. I will update the num partitions to 3 here, 1 might be sufficient, but having more than consumers does not hurt.